xml/System.Security.Cryptography.Xml/DSAKeyValue.xml

```diff
@@ -262,7 +262,15 @@
  ]]></format>
         </remarks>
         <exception cref="T:System.ArgumentNullException">The <paramref name="value" /> parameter is <see langword="null" />.</exception>
-        <exception cref="T:System.Security.Cryptography.CryptographicException">The <paramref name="value" /> parameter is not a valid <see cref="T:System.Security.Cryptography.Xml.DSAKeyValue" /> XML element.</exception>
+        <exception cref="T:System.Security.Cryptography.CryptographicException">The <paramref name="value" /> parameter is not a valid <see cref="T:System.Security.Cryptography.Xml.DSAKeyValue" /> XML element.
+
+-or-
+
+The XML has the incorrect schema.
+
+-or-
+
+The DSA parameters are invalid.</exception>
```

Comment:
The two inserted ones seem to be replacing/expanding the existing sentence (saying when it would be invalid).  I'd either remove the current sentence or not add the two new ones; whichever feels like a better balance of succinctness and friendliness.

Reply:
I think we can keep the original one. The tool ported these additional exception messages because they looked very different than the original, but thanks for clarifying that they are the same. I'll do the same with the other case.